elife_coverletter.md

```diff
@@ -0,0 +1,26 @@
+Dear Colleagues,
+
+Please consider the attached manuscript entitled, "A community maintained standard library of population genetic
+models" for publication in *elife*. This manuscript describes the first product of the PopSim Consortium, the
+stdpopsim library, which provides a game-changing resource to the population and statistical genetics community. 
+
+Simulation is a cornerstone tool in population and statistical genetics, wherein genomes or portions thereof
+are simulated for specific genomes under assumed evolutionary and demographic histories. To date researchers in these
```

Comment:
"genomes ... are simulated for specific genomes" is a bit maclunkey. Maybe change to "are simulated for specific species/populations."

Reply:
lol